zebra-chain/src/transaction.rs

```diff
@@ -174,8 +194,26 @@ impl Transaction {
                     .joinsplits()
                     .flat_map(|joinsplit| joinsplit.nullifiers.iter()),
             ),
+            // Maybe JoinSplits, maybe not, we're still deciding
+            Transaction::V5 { .. } => {
+                unimplemented!(
+                    "v5 transaction format as specified in ZIP-225 after decision on 2021-03-05"
```

Comment:
I got the arborist date wrong:
```suggestion
                    "v5 transaction format as specified in ZIP-225 after decision on 2021-03-12"
```